src/components/OnePerson.tsx

```diff
@@ -0,0 +1,71 @@
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  people: Person[];
+  searchParams: URLSearchParams;
+};
+
+export const OnePerson: React.FC<Props> = ({
+  person,
+  people,
+  searchParams,
+}) => {
+  const { personSlug } = useParams();
+  const selectedPerson = personSlug ? personSlug : '';
+
+  function findParentLink(name: string | null) {
```

Comment:
It is important to stick to one way of creating functions, fix it everywhere.
```suggestion
  const findParentLink = (name: string | null) => {
```